third_party/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -2730,19 +2826,396 @@ impl Frame {
         }
     }
 
-    fn resolver<'a>(
-        &self,
-        module_storage: &'a impl ModuleStorage,
-        resource_resolver: &'a impl ResourceResolver,
-    ) -> Resolver<'a> {
-        self.function
-            .get_resolver(module_storage, resource_resolver)
-    }
-
     fn location(&self) -> Location {
         match self.function.module_id() {
             None => Location::Script,
             Some(id) => Location::Module(id.clone()),
         }
     }
+
+    pub(crate) fn constant_at(&self, idx: ConstantPoolIndex) -> &Constant {
```

Comment:
It could be very good to start breaking down this file a bit. We can separate impl blocks based on the visibility, i.e. have pub(crate) / pub / private interfaces separately.

Reply:
Moved to new file - all these were coming from resolver so they are `pub(crate)`